xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,239 @@
+     <literal>Das U-Boot</literal>, which is loaded before the operating system
+     and can optionally show a boot menu, like GRUB on &x86; hardware.
+    </para>
+   </listitem>
+  </orderedlist>
+  
+  <para>
+   Installation can be performed from a bootable USB device, such as a memory
+   stick or USB optical drive. The exact procedure may differ from model to
+   model, depending on how the individual device is configured.
+  </para>
+  <para>
+   The &rpi; 3 Model B+ is shipped pre-configured to boot from USB devices and
+   no additional steps should be required.
+  </para>
+  <para>
+   Depending on whether this has previously been done on a particular unit, the
+   older &rpi; 3 Model B may require additional configuration to boot from USB.
+   The procedure is described on the &rpi; Foundation web site, here:
+   <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/msd.md"/>
+  </para>
+  
+ </sect2>
+ <sect2>
+  <title>
+   Network Booting the &rpi;
+  </title>
+  <para>
+   Due to the hardware's lack of on-board firmware, network-booting the &rpi;
+   using PXE is more complex than with more conventional computers.   
+  </para>
+  <para>
+   At the time of writing, network-booting &sle; on &rpi; hardware still has the 
+   status of work in progress.
+  </para>
+  <para>
+   The &rpi; Foundation publishes information on how to PXE boot one &rpi; from
+   another &rpi;, here: 
+   <link xlink:href="https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/net_tutorial.md"/>
+  </para>
+ </sect2>
+ <sect2>
+  <title>
+   See Also
+  </title>
+  <variablelist>
+   <varlistentry>
+    <term>
+     &sle; &product-ga;.&product-sp; Release Notes
+    </term>
+    <listitem>
+     <para>
+      For more information about hardware compatibility, supported options and
+      functionality when running on &rpi; hardware, please consult the relevant
+      section of the Release Notes:  
+     </para>
+     <para>
+      <link xlink:href="https://www.suse.com/releasenotes/aarch64/SUSE-SLES/15-SP1/#fate-325731"/>
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     OpenSUSE Hardware Compatibility List: Raspberry Pi3 
+    </term>
+    <listitem>
+     <para>
+      The openSUSE Project also has information about installing and configuring
+      &rpi; hardware. Much of this also applies to &sle;.
+     </para>
+     <para>
+      <link xlink:href="https://en.opensuse.org/HCL:Raspberry_Pi3"/>
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term>
+     Das U-Boot
+    </term>
+    <listitem>
+     <para>More information about <literal>Das U-Boot</literal> bootloader can
+      be found on the project's GitHub page: 
+     </para>
+     <para>
+      <link xlink:href="https://github.com/u-boot/u-boot"/>
+     </para>     
+    </listitem>
+   </varlistentry>
+  </variablelist>
```

Comment:
I'm missing a reference to our existing quick-start documentation section anywhere here.